docs/copilot/copilot-chat.md

```diff
@@ -57,6 +57,12 @@ The following example shows the `preview` mode for inline chat.
 
 ![Copilot inline chat preview mode, showing the code changes side-by-side](images/copilot-chat/inline-chat-preview-mode.png)
 
+## Experimental terminal inline chat
+
+Inline chat for the terminal can be enbled with `terminal.integrated.experimentalInlineChat` and invoked in a terminal (`kb(inlinechat.start)`).
+
+The chat will use the ![@terminal](###terminal) chat participant, which has context about the integrated terminal shell and its contents.
```

Comment:
I wanted to do `![`@terminal`]` here but it didn't work in the preview...

Reply:
The `!` prefix means it's an image, that's probably what was causing the issue.